test/browser/browser_asrouter_cfr.js

```diff
@@ -67,3 +75,38 @@ add_task(async function test_cfr_notification_show() {
   Assert.equal(PopupNotifications._currentNotifications.length, 0,
     "Should have removed the notification");
 });
+
+add_task(async function test_cfr_addon_install() {
+  // Prevent fetching the real addon url and making a network request
+  CFRPageActions._maybeAddAddonInstallURL = x => x;
```

Comment:
We'll probably want this at the top level of this file and restore original functionality if none of the tests here would trigger install to avoid breaking other tests that might rely more on parts of the actual implementation.